debug-ui/scripts/stats-server.ts

```diff
@@ -130,6 +130,11 @@ export class ManifestStatsServer {
       ssl: { rejectUnauthorized: false }, // May be needed depending on Fly Postgres configuration
     });
 
+    this.pool.on('error', (err) => {
```

Comment:
I think this is the best strategy to address the db crash issue. Others are mitigations so can be helpful, but this is what I think is best part